### PR TITLE
feat(communications): add notice component

### PR DESCRIPTION
**Pull request by yuvarajuv**
image
image

### CODE REVIEW

File: react/src/Communications/Notice/Notice.js [hunk 1]
```diff
@@ -1,7 +1,39 @@
...
+        <div className="ml-auto">
+          <svg
+            className={`w-4 h-4 transition-transform ${isCollapsed ? 'rotate-0' : 'rotate-180'}`}
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke="currentColor"
+          >
+            <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M19 9l-7 7-7-7" />
+          </svg>
```
Comment (send2alamelu) [inline react/src/Communications/Notice/Notice.js:24]: use TwixtIcon like below.

```
<TwixtIcon type="chevronUp" variant="outline />
<TwixtIcon type="chevronDown" variant="outline />
```

File: react/src/Communications/Notice/Notice.js [hunk 2]
```diff
@@ -1,7 +1,39 @@
...
+        </div>
+      </div>
+      {!isCollapsed && (
+        <div className="p-4">
+          <div>{children}</div>
+          <div className="mt-4">
+            {cta}
```
Comment (send2alamelu) [inline react/src/Communications/Notice/Notice.js:31]: can this shortcut avoided and keep a meaning fullname

**Review by send2alamelu — CHANGES_REQUESTED** [review]

File: react/src/Communications/Notice/Notice.js (hunk 2 above)
Comment (yuvarajuv) [reply react/src/Communications/Notice/Notice.js]: Changed prop name to `callToActionButtons`

File: react/src/Communications/Notice/Notice.js (hunk 1 above)
Comment (yuvarajuv) [reply react/src/Communications/Notice/Notice.js]: Done

**Pull request merged**